.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: node_js
+node_js:
+- iojs
+- '0.12'
+- '0.10'
```

Comment:
Should'nt we have later versions of node? Like add at least 5 and 6. If we include 4 we should get ride of iojs. @lukasfroehlich1